core/Types.h

```diff
@@ -755,6 +755,9 @@ TYPE(MetaType) final {
 
     TypePtr _approximate(const GlobalState &gs, const TypeConstraint &tc) const;
     TypePtr underlying(const GlobalState &gs) const;
+
+private:
+    bool canCallNew(const GlobalState &gs) const;
```

Comment:
This is the only private helper method in all of `Types.h`. Can you move this to being a function in an anonymous namespace in `calls.cc` like all other helper functions?